fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -4102,6 +4102,9 @@ public void testConcurrentPutsWithPairtrees() throws InterruptedException, IOExc
     }
 
     @Test
+    // TODO Test flaps in travis, causing more PUTs to succeed some of the time, possibly
+    // because the etag is timestamp based. Should also make sure the correct failure status returned
+    @Ignore
```

Comment:
This seems unrelated to the current work.

Reply:
It is, but the test is failing all builds. It passed in the original commit that enabled it though, and passes locally. I could open a separate PR if you prefer.

Reply:
I do not like creating unnecessary work... but if this commit gets squashed with the work you are doing on build directories, it may be an odd mix when analyzing in the future.

If we are going to turn off the test, should we have a JIRA (linked in the @Ignore annotation) as a reminder to turn it back on?

Reply:
I've moved it over to https://github.com/fcrepo4/fcrepo4/pull/1623 and added a note about it to https://jira.lyrasis.org/browse/FCREPO-2023